### ISSUE TITLE
Easy Effects does not launch on Manjaro

### ISSUE THREAD

**Issue by mackworthy** [wwmm/easyeffects#2260]
I've tried installing via flatpak and from the [Arch repos](https://archlinux.org/packages/community/x86_64/easyeffects/). I don't get any output or errors from either - it just hangs.

The `G_MESSAGES_DEBUG` flag produces output on the Arch repo version, but not the flatpak.

Arch repo version debug output:
```
$ G_MESSAGES_DEBUG=com.github.wwmm.easyeffects easyeffects 
(process:37984): easyeffects-DEBUG: 10:12:17.607: 	easyeffects.cpp:35	easyeffects version: 7.0.3
(process:37984): easyeffects-DEBUG: 10:12:17.610: 	easyeffects.cpp:45	locale directory: /usr/share/locale
(easyeffects:37984): easyeffects-DEBUG: 10:12:17.616: 	presets_manager.cpp:49	system input presets directory: "/etc/easyeffects/input"; 
(easyeffects:37984): easyeffects-DEBUG: 10:12:17.616: 	presets_manager.cpp:49	system input presets directory: "/etc/xdg/easyeffects/input"; 
(easyeffects:37984): easyeffects-DEBUG: 10:12:17.616: 	presets_manager.cpp:52	system output presets directory: "/etc/easyeffects/output"; 
(easyeffects:37984): easyeffects-DEBUG: 10:12:17.616: 	presets_manager.cpp:52	system output presets directory: "/etc/xdg/easyeffects/output"; 
(easyeffects:37984): easyeffects-DEBUG: 10:12:17.616: 	presets_manager.cpp:194	user presets directory already exists: /home/reg/.config/easyeffects/
(easyeffects:37984): easyeffects-DEBUG: 10:12:17.616: 	presets_manager.cpp:194	user presets directory already exists: /home/reg/.config/easyeffects/input
(easyeffects:37984): easyeffects-DEBUG: 10:12:17.616: 	presets_manager.cpp:194	user presets directory already exists: /home/reg/.config/easyeffects/output
(easyeffects:37984): easyeffects-DEBUG: 10:12:17.616: 	presets_manager.cpp:194	user presets directory already exists: /home/reg/.config/easyeffects/autoload/input
(easyeffects:37984): easyeffects-DEBUG: 10:12:17.616: 	presets_manager.cpp:194	user presets directory already exists: /home/reg/.config/easyeffects/autoload/output
(easyeffects:37984): easyeffects-DEBUG: 10:12:18.075: 	pipe_manager.cpp:1427	compiled with PipeWire: 0.3.67
(easyeffects:37984): easyeffects-DEBUG: 10:12:18.075: 	pipe_manager.cpp:1428	linked to PipeWire: 0.3.68
(easyeffects:37984): easyeffects-DEBUG: 10:12:18.091: 	pipe_manager.cpp:1398	core version: 0.3.67
(easyeffects:37984): easyeffects-DEBUG: 10:12:18.092: 	pipe_manager.cpp:1399	core name: pipewire-0
(easyeffects:37984): easyeffects-DEBUG: 10:12:18.092: 	pipe_manager.cpp:1312	found metadata: settings
```

**Comment by glspdotnet** [comment]
I'm experiencing the same issue on a fresh Manjaro (w/KDE) install. I installed using Pamac via the official repositories with all optional dependencies.

This is my output after killing all instances and running manually:

```bash
$ G_MESSAGES_DEBUG=easyeffects easyeffects
(process:3542): easyeffects-DEBUG: 21:03:52.666:        easyeffects.cpp:35      easyeffects version: 7.0.3
(process:3542): easyeffects-DEBUG: 21:03:52.667:        easyeffects.cpp:45      locale directory: /usr/share/locale
(easyeffects:3542): easyeffects-DEBUG: 21:03:52.668:    presets_manager.cpp:49  system input presets directory: "/etc/easyeffects/input"; 
(easyeffects:3542): easyeffects-DEBUG: 21:03:52.668:    presets_manager.cpp:49  system input presets directory: "/etc/xdg/easyeffects/input"; 
(easyeffects:3542): easyeffects-DEBUG: 21:03:52.668:    presets_manager.cpp:49  system input presets directory: "/home/john/.config/kdedefaults/easyeffects/input"; 
(easyeffects:3542): easyeffects-DEBUG: 21:03:52.668:    presets_manager.cpp:52  system output presets directory: "/etc/easyeffects/output"; 
(easyeffects:3542): easyeffects-DEBUG: 21:03:52.668:    presets_manager.cpp:52  system output presets directory: "/etc/xdg/easyeffects/output"; 
(easyeffects:3542): easyeffects-DEBUG: 21:03:52.668:    presets_manager.cpp:52  system output presets directory: "/home/john/.config/kdedefaults/easyeffects/output"; 
(easyeffects:3542): easyeffects-DEBUG: 21:03:52.668:    presets_manager.cpp:194 user presets directory already exists: /home/john/.config/easyeffects/
(easyeffects:3542): easyeffects-DEBUG: 21:03:52.668:    presets_manager.cpp:194 user presets directory already exists: /home/john/.config/easyeffects/input
(easyeffects:3542): easyeffects-DEBUG: 21:03:52.668:    presets_manager.cpp:194 user presets directory already exists: /home/john/.config/easyeffects/output
(easyeffects:3542): easyeffects-DEBUG: 21:03:52.668:    presets_manager.cpp:194 user presets directory already exists: /home/john/.config/easyeffects/autoload/input
(easyeffects:3542): easyeffects-DEBUG: 21:03:52.668:    presets_manager.cpp:194 user presets directory already exists: /home/john/.config/easyeffects/autoload/output
(easyeffects:3542): easyeffects-DEBUG: 21:03:52.765:    pipe_manager.cpp:1427   compiled with PipeWire: 0.3.67
(easyeffects:3542): easyeffects-DEBUG: 21:03:52.765:    pipe_manager.cpp:1428   linked to PipeWire: 0.3.68
(easyeffects:3542): easyeffects-DEBUG: 21:03:52.773:    pipe_manager.cpp:1398   core version: 0.3.68
(easyeffects:3542): easyeffects-DEBUG: 21:03:52.773:    pipe_manager.cpp:1399   core name: pipewire-0
(easyeffects:3542): easyeffects-DEBUG: 21:03:52.773:    pipe_manager.cpp:1312   found metadata: settings
(easyeffects:3542): easyeffects-DEBUG: 21:03:52.773:    pipe_manager.cpp:1312   found metadata: default
(easyeffects:3542): easyeffects-DEBUG: 21:03:52.773:    pipe_manager.cpp:1312   found metadata: route-settings
```

At that point the process becomes unresponsive and needs to be killed externally.

[`pw-dump` output](https://github.com/wwmm/easyeffects/files/11227803/pw-dump.txt)

**Comment by wwmm** [comment]
>At that point the process becomes unresponsive and needs to be killed externally.

Last week someone had the same issue #2243 and the reason was also a problem in the PipeWire installation.

**Comment by wwmm** [comment]
>pulseaudio-zeroconf is an optional dependency for manjaro-pulse and I didn't want to take a chance on causing an issue by removing it if it's something that Manjaro expects to be there. (I don't want to break my new OS install right off the bat. sweat_smile)

You can remove it. Zeroconf will use `avahi` to to play sound through the network. Your system will be fine without it. And even if you want to do something like this PipeWire probably has its own way to do it.

**Comment by wwmm** [comment]
As I use Arch Linux and not Manjaro I do not know exactly what `manjaro-pulse` is. But you can't have Pulseaudio and PipeWire at the same time when PipeWire is managing audio. One of them has to go away.

**Comment by glspdotnet** [comment]
> As I use Arch Linux and not Manjaro I do not know exactly what `manjaro-pulse` is. But you can't have Pulseaudio and PipeWire at the same time when PipeWire is managing audio. One of them has to go away.

Thanks for the super-quick responses! Yeah, `manjaro-pulse` is just the OS's meta-package that handles installation of Pulse & all of its related dependencies. I installed `manjaro-pipewire` (which then replaced the system-wide Pulse-related dependencies with those of PipeWire) and restarted and everything's now working great! If I was more up-to-date in my Linux knowledge I would have gone with PipeWire when I installed the OS but I instead went with what I knew, as it's been a while since I've installed Linux as a desktop OS.

@mackworthy Sorry for accidentally hijacking your thread! 😅 If you haven't yet tried replacing Pulse with PipeWire, maybe give that a shot if you don't have anything else that requires it.
